lib/qam.pm

```diff
@@ -59,7 +59,26 @@ sub snap_revert {
     my ($svirt, $vm_name, $snapshot) = @_;
     my $ret = $svirt->run_cmd("virsh snapshot-revert $vm_name $snapshot --running");
     die "Snapshot revert $snapshot failed" if $ret;
+}
+
+sub zypper_call_log {
```

Comment:
this is a full copy of the zypper_call function with a minor difference.  Please integrate this feature into the zypper_call function, possibly by using named parameters